data_set/standard/extract.md

```diff
@@ -75,6 +75,8 @@ Extract a **data set profile's** data.
 #### ❌ Raises
 * `SecurityRequestError`<br>
   Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+* `SegmentError`<br>
+  Raises `SegmentError` when the list of **segments** supplied contains at least one unknown segment.
```

Comment:
```markdown
* `SegmentError`<br>
  Raises `SegmentError` when the list of **segments** provided contains at least one **unknown** segment.
```

Make the same updates everywhere else this exception is documented.

Reply:
Change made.

Reply:
```markdown
* `SegmentError`<br>
  Raises `SegmentError` when the list of **segments** provided contains one or more **unknown** segments.
```

Reply:
Fair enough, change made.